src/data/contributors/Dev.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Dev Batra",
+  "skills": ["JAVA", "PYTHON","HTML","CSS"],
+  "cause": "Coding For The Community",
```

Comment:
code for is already the part of string, please only write the cause.

Also, please write github profile link, if you don't have twitter or linkedIn links.

Reply:
Sure sir I'll be resolving the issues soon.